deps/http_parser/test.c

```diff
@@ -3356,7 +3356,7 @@ test_double_content_length_error (int req)
 
   parsed = http_parser_execute(&parser, &settings_null, buf, buflen);
   if (parsed != buflen) {
-    assert(HTTP_PARSER_ERRNO(&parser) == HPE_MULTIPLE_CONTENT_LENGTH);
+    assert(HTTP_PARSER_ERRNO(&parser) == HPE_UNEXPECTED_CONTENT_LENGTH);
```

Comment:
It didn't. This was caught after. We don't run the http-parser tests in our
main build so this snuck through. It was fixed upstream so making sure it's
fixed here too.
On Feb 15, 2016 9:51 AM, "Ben Noordhuis" <notifications@github.com> wrote:

> In deps/http_parser/test.c
> <https://github.com/nodejs/node/pull/5237#discussion_r52927301>:
>
> > @@ -3356,7 +3356,7 @@ test_double_content_length_error (int req)
> >
> >    parsed = http_parser_execute(&parser, &settings_null, buf, buflen);
> >    if (parsed != buflen) {
> > -    assert(HTTP_PARSER_ERRNO(&parser) == HPE_MULTIPLE_CONTENT_LENGTH);
> > +    assert(HTTP_PARSER_ERRNO(&parser) == HPE_UNEXPECTED_CONTENT_LENGTH);
>
> Do I read right that HPE_MULTIPLE_CONTENT_LENGTH is not returned
> anywhere? How did this test pass before?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nodejs/node/pull/5237/files#r52927301>.
>